tutorial/getting-started-aaos.md

```diff
@@ -0,0 +1,51 @@
+This tutorial guides you through basic operations that you can do with an application based on an Android Automotive OS (AAOS) image. 
+
+In this tutorial, we will be specifically focusing on the following tasks:
+- Creating an application using an AAOS image
+- Stream the application you created with a specific screen resolution
+- Set the temperature of the automotive using the Vehicle Hardware Abstraction Layer (VHAL) controls
```

Comment:
Need your opinions on usage of automotive vs automobile:

Automotive indicates any type of vehicle (could include boats, planes etc) while automobile indicates a more specific four-wheeled vehicle that is used for transportation.

I have currently used automotive since the application type on the dashboard has that option but automobile seems like a better fit since automotive generally aligns more with the industry and automobile seems to align with the vehicle itself. What do others think?

Reply:
Decided in SU to keep it as automotive.